src/Chapter2.hs

```diff
@@ -728,7 +742,7 @@ value of the element itself
 🕯 HINT: Use combination of 'map' and 'replicate'
 -}
 smartReplicate :: [Int] -> [Int]
-smartReplicate l = error "smartReplicate: Not implemented!"
+smartReplicate = concatMap (\x->replicate x x)
```

Comment:
Great usage of standard functions! 👍🏻

Reply:
thanks to the linter (it was `concat $ map .....`) :P didn't know flatmap is called a concatMap in haskell